test/helpers/test-case.js

```diff
@@ -0,0 +1,27 @@
+const header = 'const test = require(\'ava\');';
+
+function testCaseBuilder(ruleId, options = []) {
```

Comment:
Needs a doc comment on what it does and how to use it.

Reply:
I tend to use [jsDoc](https://jsdoc.app/about-getting-started.html) for this. Do you prefer something else ?

Reply:
jsDoc is fine, but I think the types are. Really just needs a normal comment explaining it.